ios/PaymentSheetAppearance.swift

```diff
@@ -340,13 +335,32 @@ internal class PaymentSheetAppearance {
             return defaultColor
           })
       }
-    
+
```

Comment:
Nit: Include a comment describing the function similar to Android

```suggestion
/**
 * Parses a themedColor from [params] at [key]. Supports both:
 * - Single hex string: "#RRGGBB"
 * - Light/dark object: { "light": "#RRGGBB", "dark": "#RRGGBB" }
 * For light/dark objects, chooses the appropriate color based on current UI mode.
 * Falls back to [defaultColor] if no color is provided.
 */
```